modules/integration_azure-event-hub/conf/01-capacity-throttling.yaml

```
@@ -0,0 +1,23 @@
+
+---
+module: "Azure Event Hub"
+name: "throttled requests"
+filtering: "filter('resource_type', 'Microsoft.EventHub/clusters') and filter('primary_aggregation_type', 'true')"
+aggregation: ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+transformation: ".max(over='15m')"
+signals:
+  A:
+    metric: ThrottledRequests
+  B:
+    metric: IncomingRequests
+  signal:
+    formula: "(A/B).scale(100).fill(0)"
+rules:
+  critical:
+    threshold: 90
+    comparator: ">"
+  major:
+    threshold: 80
```

```suggestion
    threshold: 0
```
We should be warned if we have any throttled request.